src/bootstrap/bootstrap.py

```diff
@@ -495,81 +495,6 @@ def download_toolchain(self, stage0=True, rustc_channel=None):
                 with output(self.rustfmt_stamp()) as rustfmt_stamp:
                     rustfmt_stamp.write(self.stage0_rustfmt.channel())
 
-        # Avoid downloading LLVM twice (once for stage0 and once for the master rustc)
-        if self.downloading_llvm() and stage0:
```

Comment:
I figured it was better to do this all at once, rather than maintaining duplicate code - this way, we'll see error reports immediately, instead of getting a whole bunch of reports for separate features at once when people start using `cargo run` instead of x.py.